src/pybind/mgr/dashboard/frontend/src/app/ceph/smb/smb-cluster-form/smb-cluster-form.component.html

```diff
@@ -0,0 +1,343 @@
+    </ng-container>
+
+    <div class="form-item"
+         *ngIf="this.smbForm.get('auth_mode').value === 'user'">
+      <button cdsButton="tertiary"
+              type="button"
+              (click)="addUserGroupSetting()"
+              i18n>
+        Add User Group Id
+        <svg cdsIcon="add"
+             size="32"
+             class="cds--btn__icon"
+             icon></svg>
+      </button>
+    </div>
+
+    <!-- Placement -->
+    <ng-container *ngIf="orchStatus.available">
+      <div class="form-item">
+        <cds-select
+          label="Placement"
+          for="placement"
+          formControlName="placement"
+          name="placement"
+          id="placement"
+          i18n
+        >
+          <option value="hosts">Hosts</option>
+          <option value="label">Labels</option>
+        </cds-select>
+      </div>
+      <ng-container *ngIf="hostsAndLabels$ | async as data">
+        <!-- Label -->
+        <div *ngIf="smbForm.controls.placement.value === 'label'"
+             class="form-item">
+          <cds-combo-box
+            type="multi"
+            selectionFeedback="top-after-reopen"
+            label="Label"
+            for="label"
+            name="label"
+            formControlName="label"
+            id="label"
+            placeholder="Select labels..."
+            [appendInline]="true"
+            [items]="data.labels"
+            i18n-placeholder
+            (selected)="multiSelector($event, 'label')"
+            [invalid]="smbForm.controls.label.invalid && smbForm.controls.label.dirty"
+            [invalidText]="labelError"
+            cdRequiredField="Label"
+            i18n
+          >
+            <cds-dropdown-list></cds-dropdown-list>
+          </cds-combo-box>
+          <ng-template #labelError>
+            <span
+              class="invalid-feedback"
+              *ngIf="smbForm.showError('label', formDir, 'required')"
+              i18n
+              >This field is required.</span
+            >
+          </ng-template>
+        </div>
+
+        <!-- Hosts -->
+        <div *ngIf="smbForm.controls.placement.value === 'hosts'"
+             class="form-item">
+          <cds-combo-box
+            type="multi"
+            selectionFeedback="top-after-reopen"
+            label="Hosts"
+            for="hosts"
+            name="hosts"
+            formControlName="hosts"
+            id="hosts"
+            placeholder="Select hosts..."
+            i18n-placeholder
+            [appendInline]="true"
+            [items]="data.hosts"
+            (selected)="multiSelector($event, 'hosts')"
+            i18n
+          >
+            <cds-dropdown-list></cds-dropdown-list>
+          </cds-combo-box>
+        </div>
+      </ng-container>
+    </ng-container>
+
+    <div class="form-item">
+      <cds-number
+        [id]="'count'"
+        [name]="'count'"
+        [formControlName]="'count'"
+        [label]="'Count'"
+        [min]="1"
+        i18n
```

Comment:
```suggestion
```
I think i18n would not work here. Do we have something for labels ? cc @nizamial09